components/style/properties/data.py

```diff
@@ -121,9 +121,7 @@ def __init__(self, style_struct, name, spec=None, animatable=None, derived_from=
         self.alias = alias.split() if alias else []
         self.extra_prefixes = extra_prefixes.split() if extra_prefixes else []
         self.boxed = arg_to_bool(boxed)
-        self.creates_stacking_context = arg_to_bool(creates_stacking_context)
-        self.fixpos_cb = arg_to_bool(fixpos_cb)
-        self.abspos_cb = arg_to_bool(abspos_cb)
+        self.flags = flags.split() if flags else []
```

Comment:
Well, when I try to default flags argument to `[]`, python gives an AttributeError that tells split is not an attribute of `list`.